src/components/Species/index.tsx

```diff
@@ -152,8 +153,13 @@ export default function SpeciesList({ reloadData, species }: SpeciesListProps):
       return '';
     }
   };
-  const columns: TableColumnType[] = React.useMemo(
-    () => [
+  const columns: TableColumnType[] = React.useMemo(() => {
```

Comment:
We could remove the useMemo and see if it really affects performance.
Another option is to maybe pass the strings into the useMemo function - expose the strings in useLocalization, that perhaps negates the benefit of loading strings from `/src/strings`.
For now this is harmless and works.

Reply:
Removing `useMemo` makes the compiler complain that the `useEffect` call that declares `columns` as a dependency will be flagged as changed on every render.

Reply:
Oops, yes that won't work.